.github/workflows/cleanup.yml

```diff
@@ -0,0 +1,19 @@
+name: cleanup
+
+on:
+  schedule:
+    - cron: '30 5 * * 1'
+  workflow_dispatch:
+  push:
+    branches:
+      - 'aka/cleanup_prs'
```

Comment:
Does this need to be removed prior to merge?

Reply:
Correct. Removed